inputs.css

```diff
@@ -0,0 +1,29 @@
+.form-input-size {
+    width: 180px;
+}
+
+.search-input-size {
+    width: 800px;
+}
+
+.text-input {
+    background-color: var(--white);
+    border: 1px solid var(--gray-3);
+    padding-top: 0.5rem;
+    padding-bottom: 0.5em;
+    border-radius: 0.3em;
+    padding-left: 0.5em;
+    font-size: 0.8em;
+}
+
+.text-input-with-icon {
+    padding-left: 2em;
+}
+
+.text-input-with-icon svg {
```

Comment:
This should have `position: absolute` here. This way, we only need to add this class and it will work. Without having to add aditional classes in the HTML